documents/translation-workflow.md

```diff
@@ -0,0 +1,79 @@
+## Translation Workflow
+
+The purpose of this document is to explain the general workflow and commands required for our translation process on Deriv.com
+
+---
+
+### General workflow will be as such:
+⚠️ Note: If you're not managing the translations on Crowdin, the workflow usually stops at step #3.
+
+<br />
+
+1. PR will be based on crowdin branch
+
+    - if you dont have crowdin branch
+
+    ```sh
+    git fetch upstream crowdin:crowdin
+    git checkout crowdin
+    git checkout -b [_your_branch_name]
+    ```
+
+    -   if you have crowdin branch
+
+    ```sh
+    git checkout crowdin
+    git pull upstream crowdin
+    git checkout -b [_your_branch_name]
+    ```
+
+2. Make your changes and submit for review followed by checking by QA. Only the `English version` is checked at this staged by QA as the new strings have not been extracted for translation purposes.
+
+3. Once it passes QA, it will be merged into the Crowdin branch.
+
+<br />
+
+### Extracting & Managing Translations:
+
+4. Extract translations from Crowdin branch
```

Comment:
step 7 can be combined with step 4

downloading translation will have code change that needs to be committed as well

Reply:
As discussed, this was a misunderstanding and steps 4 & 7 will remain separated.